projects/packages/forms/tests/php/contact-form/Contact_Form_Test.php

```diff
@@ -747,7 +747,7 @@ public function test_array_values_with_commas_and_brackets_from_gutenblock() {
 			'options' => array( '"foo"', 'bar, baz', '[b\\rackets]' ),
 			'label'   => 'fun ][ times',
 		);
-		$html = Contact_Form_Plugin::gutenblock_render_field_radio( $attr, '' );
+		$html = Contact_Form_Plugin::gutenblock_render_field_radio( $attr, '', null );
```

Comment:
Just to suppress a PHP warning about expected args

Reply:
I just noticed that Phan is is complaining about the `null` argument being passed in here due to the function signature of `gutenblock_render_field_radio`. Can we pass it a `new WP_Block()` of some kind?

Reply:
Just tried a tiny commit to see if it'll fix this one up: https://github.com/Automattic/jetpack/pull/43002/commits/db1f0af8076e81de14b832a3513162430317a2d7

Reply:
Good one, thanks! 🙇🏻